llarp/router/outbound_message_handler.cpp

```diff
@@ -86,7 +84,7 @@ namespace llarp
     m_Killer.TryAccess([this]() {
       recentlyRemovedPaths.Decay();
       ProcessOutboundQueue();
-      if (/*bool more = */ SendRoundRobin())
```

Comment:
Without this comment it's easy to conclude that `SendRoundRobin` returns a success/fail rather than the actual purpose of its return, so I think it should stay.

Reply:
honestly the entire reason for this type is that you dont want to continue processing if you dont process it all.
we should revert that, but we'd need a lot of testing here for that. i dont feel like it is okay to do 1 week before hard fork.